### DISCUSSION TITLE
I wish there was a feature to add profile pic on my phone!

### DISCUSSION THREAD

**Discussion by kathylopez97** [community/community#118618] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.157.0 (127735376)
Device: iPhone 13 Pro Max (iOS 17.5)
</details>

**Comment by eliperkins** [comment]
Hey @kathylopez97 , thanks for this feedback! Updating profile pictures from GitHub Mobile seems like a natural feature for the app. I've added your notes to an internal tracking issue for this feature, and I'll let you know when the team has picked this up.

Thanks for helping improve GitHub Mobile!

**Reply by kathylopez97** [reply]
Thank you for your response. @eliperkins 
I appreciate your acknowledgement. I look forward to more additional features on Github moblie. 😊
